src/Microsoft.DotNet.Wpf/test/System.Xaml/System.Xaml.Tests.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Configurations>netcoreapp-Debug;netcoreapp-Release</Configurations>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\System.Xaml\System.Xaml.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="2.4.0">
```

Comment:
what is this package reference for? do other dotnet repos use this?

Reply:
Yeah - its used for code coverage /cc @ViktorHofer